docs/src/analysis/sensitivity.md

```diff
@@ -383,3 +385,185 @@ end
+
+`regre_sensitivity = regression_sensitivity(prob::DiffEqBase.DEProblem,alg,t,param_range,param_fixed,n;coeffs=:rank)`
+
+Again, `f` and `param_range` are the same as above. An array of the true parameter values 
+that lie within the `param_range` bounds are passed through the `param_fixed` argument. 
+`n` determines the number of simulations of the model run to generate the data points 
+of the solution and parameter values and the `coeffs` kwarg lets you decide the
+coefficients you want.
+
+### GSA example 
+
+Let's create the ODE problem to run our GSA on.
+
+```julia
+f = @ode_def_nohes LotkaVolterraTest begin
+    dx = a*x - b*x*y
+    dy = -3*y + x*y
+end a b 
+u0 = [1.0;1.0]
+tspan = (0.0,10.0)
+p = [1.5,1.0]
+prob = ODEProblem(f,u0,tspan,p)
+t = collect(range(0, stop=10, length=200))
+```
+For Morris Method
+
+```julia
+m = DiffEqSensitivity.morris_sensitivity(prob,Tsit5(),t,[[1,5],[0.5,5]],[10,10],len_trajectory=1500,total_num_trajectory=1000,num_trajectory=150)
+```
+Let's get the means and variances from the `MorrisSensitivity` struct.
+
+```julia
+m.means
+
+Out[9]: 2-element Array{Array{Float64,2},1}:
+ [0.0 0.0513678 … 7.91336 7.93783; 0.0 0.00115769 … 3.66156 3.67284]
+ [0.0 0.0488899 … 2.50728 2.359; 0.0 0.00112006 … 2.23431 2.44946]  
+
+m.variances
+
+Out[10]: 2-element Array{Array{Float64,2},1}:
+ [0.0 1.94672e-5 … 26.4223 24.8513; 0.0 4.81347e-9 … 37.4061 30.3068]
+ [0.0 1.77615e-5 … 17.9555 14.9231; 0.0 4.47931e-9 … 48.074 51.9312] 
+```
+This gives the means of the effects and it's variances over the entire timespan and thus we get 200-length 
+arrays for each paramter and dependent variable pair.
+
+We can plot the trajectory of the sensitivity with the standard deviation bars.
+```julia
+# For the first parameter (a)
+stdv1 = sqrt.(m.variances[1])
+p = plot(m.means[1]', yerror=stdv1)
+```
+![morrisparameter1](../assets/morris1.png)
+
+```julia
+# For the second parameter (b)
+stdv2 = sqrt.(m.variances[2])
+p = plot(m.means[2]', yerror=stdv2)
+```
+![morrisparameter2](../assets/morris2.png)
+
+For Sobol Method
+
+```julia
+
+s0 = sobol_sensitivity(prob,Tsit5(),t,[[1,5],[0.5,5]],N,0)
+Out[8]: 2-element Array{Array{Float64,2},1}:
+ [NaN 0.507831 … 1.00731 1.00436; NaN 1.92336 … 0.732384 0.730945]  
+ [NaN 0.47214 … 0.676224 0.681525; NaN -1.68656 … 0.879557 0.877603]
+
+s1 = sobol_sensitivity(prob,Tsit5(),t,[[1,5],[0.5,5]],N,1)
+Out[9]: 2-element Array{Array{Float64,2},1}:
+ [NaN 0.39537 … 0.341697 0.343645; NaN -2.06101 … 0.10922 0.106976]     
+ [NaN 0.652815 … 0.00910675 0.00815206; NaN 5.24832 … 0.296978 0.296639]
+
+s2 = sobol_sensitivity(prob,Tsit5(),t,[[1,5],[0.5,5]],N,2)
+Out[10]: 1-element Array{Array{Float64,2},1}:
+ [NaN -0.0596478 … 0.652303 0.657847; NaN -1.84504 … 0.645139 0.620036]
+```
+We can decide which order of Sobol Indices we are interested in my passing an argument for it, 
+by default it gives the second order indices. Again the result is obtained over the entire `timespan`
```

Comment:
Any demonstration of the regression method?

Reply:
https://github.com/JuliaStats/GLM.jl/pull/242 regression doesn't work yet on 0.7/1.0 so I have not added it, I will do it with another PR once it starts working?

Reply:
Sounds good to me.